ci-operator/step-registry/hypershift/mce/install/hypershift-mce-install-commands.sh

```diff
@@ -120,45 +119,47 @@ for ((i=1; i<=20; i++)); do
   echo "Waiting for CSV to be ready"
 done
 
-if [ $_apiReady -eq 1 ]; then
-  # Enable Hypershift Preview
-  oc apply -f - <<EOF
+if [ $_apiReady -eq 0 ]; then
+  echo "multiclusterengine subscription could not install in the allotted time."
+  exit 1
+fi
+echo "multiclusterengine installed successfully"
+
+# Enable Hypershift Preview
+oc apply -f - <<EOF
 apiVersion: multicluster.openshift.io/v1
 kind: MultiClusterEngine
 metadata:
   name: multiclusterengine-sample
 spec: {}
 EOF
-  oc annotate mce multiclusterengine-sample imageRepository=quay.io/acm-d
-  echo "multiclusterengine installed successfully"
-  sleep 5
-else
-  echo "multiclusterengine subscription could not install in the allotted time."
-  exit 1
-fi
+sleep 5
 
 oc patch mce multiclusterengine-sample --type=merge -p '{"spec":{"overrides":{"components":[{"name":"hypershift-preview","enabled": true}]}}}'
 
 # It takes some time for this api to become available.
 # So we try multiple times until it succeeds
-# wait for hypershift operator to come online
-_localClusterReady=0
 set +e
 for ((i=1; i<=10; i++)); do
-  oc get managedcluster local-cluster -o 'jsonpath={.status.conditions[?(@.type=="ManagedClusterConditionAvailable")].status}' >> /dev/null
-  if [ $? -eq 0 ]; then
-    _localClusterReady=1
-    break
-  fi
+  oc apply -f - <<EOF
+apiVersion: cluster.open-cluster-management.io/v1
+kind: ManagedCluster
+metadata:
+  labels:
+    local-cluster: "true"
+  name: local-cluster
+spec:
+  hubAcceptsClient: true
+  leaseDurationSeconds: 60
+EOF
   echo "Waiting for MCE local-cluster to be ready..."
   sleep 15
 done
 set -e
```

Comment:
we shouldn't need a loop here anymore, or the `set +e` and `set -e`. I think this whole section can be replaced with 
```
oc apply -f - <<EOF
apiVersion: cluster.open-cluster-management.io/v1
kind: ManagedCluster
metadata:
  labels:
    local-cluster: "true"
  name: local-cluster
spec:
  hubAcceptsClient: true
  leaseDurationSeconds: 60
EOF
```